nexus/tests/integration_tests/ip_pools.rs

```diff
@@ -1142,52 +1147,94 @@ async fn test_ip_pool_range_pagination(cptestctx: &ControlPlaneTestContext) {
 #[nexus_test]
 async fn test_ip_pool_list_in_silo(cptestctx: &ControlPlaneTestContext) {
     let client = &cptestctx.external_client;
-    let mypool_name = "mypool";
 
-    const PROJECT_NAME: &str = "myproj";
-    create_project(client, PROJECT_NAME).await;
+    let silo_url = format!("/v1/system/silos/{}", cptestctx.silo_name);
+    let silo: Silo = object_get(client, &silo_url).await;
```

Comment:
None of this is particularly interesting — I'm just setting up a silo with a non-admin user and hitting IP pool list and view.